system/docker/client/Dockerfile

```diff
@@ -2,67 +2,75 @@ FROM teracy/ubuntu:16.04-dind-latest
 
 # generally useful packages
 RUN apt-get update && apt-get install -y --no-install-recommends \
-        ca-certificates \
-        apt-transport-https \
-        software-properties-common \
-        curl \
-        wget \
-        ssh \
-        vim \
+    ca-certificates \
+    apt-transport-https \
+    software-properties-common \
+    curl \
+    wget \
+    ssh \
+    vim \
     && rm -rf /var/lib/apt/lists/*
 
 # python
-ARG PYTHON3_VERSION
-ENV PYTHON3_VERSION=${PYTHON3_VERSION:-3.5}
 RUN apt-get update && apt-get install -y \
-        python${PYTHON3_VERSION} \
-        python3-pip \
-        python-setuptools \
+    python3-pip \
+    python-setuptools \
     && rm -rf /var/lib/apt/lists/*
 
 # pypi based packages
 # issues with pip>=10:
 # https://github.com/pypa/pip/issues/5240
 # https://github.com/pypa/pip/issues/5221
 RUN python3 -m pip install -U \
-        pip \
-        setuptools \
-        virtualenv==16.0.0 \
-        pipenv==2018.11.26 \
+    pip==10.0.0 \
+    setuptools \
+    virtualenv==16.0.0 \
+    pipenv==2018.11.26 \
     && pip3 list
 
-# libindy
-ARG LIBINDY_REPO_COMPONENT
-ARG LIBINDY_VERSION
-ENV LIBINDY_REPO_COMPONENT=${LIBINDY_REPO_COMPONENT:-master}
-ENV LIBINDY_VERSION=${LIBINDY_VERSION:-1.10.0~1171}
+# sovrin repo init
```

Comment:
allow greater degree of control over dockerfiles via env variables (the GHA for sovrin uses these, for example).